xray/commands/utils.go

```diff
@@ -0,0 +1,21 @@
+package commands
+
+import (
+	"github.com/jfrog/jfrog-cli-core/utils/config"
+	clientconfig "github.com/jfrog/jfrog-client-go/config"
+	"github.com/jfrog/jfrog-client-go/xray"
+)
+
+func CreateXrayServiceManager(serviceDetails *config.ServerDetails) (*xray.XrayServicesManager, error) {
```

Comment:
This is not a command, and therefore it shound't be under the commands package.
Let's move this fule  to be under xray/utils.